data_validation/cli_tools.py

```diff
@@ -168,9 +172,104 @@ def configure_arg_parser():
     _configure_find_tables(subparsers)
     _configure_raw_query(subparsers)
     _configure_beta_parser(subparsers)
+    _configure_partition_parser(subparsers)
     return parser
 
 
+def _configure_partition_parser(subparsers):
+    """Configure arguments to generate partitioned config files."""
+    partition_parser = subparsers.add_parser(
+        "generate-table-partitions",
+        help=(
+            "Generate partitions for validation and store the Config files in "
+            "a directory"
+        ),
+    )
+
+    # Group all optional arguments together
+    optional_arguments = partition_parser.add_argument_group("optional arguments")
+    optional_arguments.add_argument(
```

Comment:
We don't need threshold, use random row, or random row batch size since we aren't running a typical validation

Reply:
Done